tests/PetrolCarTests.cpp

```diff
@@ -0,0 +1,84 @@
+#include <gtest/gtest.h>
+#include "../PetrolCar.hpp"
+#include "../Gear.hpp"
+#include "../Exceptions.hpp"
+
+TEST(PetrolCarTests, 
+	ShouldHaveVelocityEqualToZeroAfterCreation)
+{
+	//TODO: prevent creating car with no engine
+	PetrolCar pc(nullptr);
+	ASSERT_EQ(pc.getVelocity(), 0);
+}
+
+TEST(PetrolCarTests, ShouldHaveNeutralGearAfterCreation)
+{
+	PetrolCar pc(nullptr);
+	ASSERT_EQ(pc.getGear(), Gear::_N);
+}
+
+TEST(PetrolCarTests, 
+	ShouldntChangeVelocityIfAccelerationIsEqualToZero)
+{
+	PetrolCar pc(nullptr);
+	auto velocityBeforeAcceleration = pc.getVelocity();
+	pc.accelerate(0);
+
+	ASSERT_EQ(velocityBeforeAcceleration, pc.getVelocity());
+}
+
+TEST(PetrolCarTests, 
+	ShouldIncreaseVelocityByGivenPositiveAccelerationValue)
+{
+	PetrolCar pc(nullptr);
```

Comment:
Bardzo fajne testy porobiliście, zgarniecie za to bonusowe punkty :)
Trochę dziwnie tylko wygląda, że taki samochód nie ma silnika, a mimo to może sobie przyspieszać. Tutaj już się czepiam, ale tylko wam wskazuję, że gdybyśmy robili taką prawdziwą funkcjonalność to na pewno przyspieszanie musiałoby wymagać obecności silnika. Ale wtedy już trzeba by było stosować mockowanie, o czym dopiero się nauczymy na zajęciach o testowaniu :)
Nie musicie tutaj nic poprawiać, to tylko luźny komentarz.